index.d.ts

```diff
@@ -1156,6 +1166,14 @@ declare namespace algoliasearchHelper {
      * total number of hits of this query on the index
      */
     nbHits: number;
+    /**
+     * total number of hits from smart sort
+     */
+    nbSortedHits?: number;
```

Comment:
We should state the word `smart` to avoid confusion with any kind of other sort, like `nbSmartSortedHits`, wdyt?

Reply:
That's a property sent by the engine, so we cannot change it :)